src/main/java/co/lotc/lever/cmd/Countdown.java

```diff
@@ -0,0 +1,37 @@
+package co.lotc.lever.cmd;
+
+import static org.bukkit.ChatColor.*;
+
+import co.lotc.core.bukkit.util.Run;
+import co.lotc.core.command.annotate.Arg;
+import co.lotc.core.command.annotate.Default;
+import co.lotc.core.command.annotate.Range;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import co.lotc.core.bukkit.util.LocationUtil;
+import co.lotc.lever.BaseCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Countdown extends BaseCommand {
+    private static final int BROADCAST_RADIUS = 20;
```

Comment:
This should shouldn't be a hard coded variable. Allow the user to set radius in the command